.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+on: [push, pull_request]
+
+jobs:
+  android-bin:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
One thing I am unclear about: For android builds, the android NDK/SDK needs to be installed as described in the setup.  It doesn't look like this CI does that.  I wouldn't have expected it to be set up in ubuntu-latest.  Also unsure if bazel needs to be installed.  Have you tested this CI?

Reply:
Both the [Android NDK](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#android) and [Bazel](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#tools) are pre-installed on the Ubuntu images!

Reply:
Oh, that's nice.  Then that explains why the linux build fails, because we need to install a specific clang/libc++ to /usr/local/lib.